### ISSUE TITLE
Numerical values passed as list of strings incorrectly treated as colors

### ISSUE THREAD

**Issue by Sija** [cocopon/tweakpane#601]
Image is worth a thousand words, so here it goes:

<img width="384" alt="image" src="https://github.com/cocopon/tweakpane/assets/988/21eed8e2-ef5f-4d9e-98db-8e9f8be02d09">

```js
const object = {
  "ISO Speed": "3200",
};
const options = {
  "100": "100",
  "125": "125",
  "160": "160",
  "200": "200",
  "250": "250",
  "320": "320",
  "400": "400",
  "500": "500",
  "640": "640",
  "800": "800",
  "1000": "1000",
  "1250": "1250",
  "1600": "1600",
  "2000": "2000",
  "2500": "2500",
  "3200": "3200",
  "4000": "4000",
  "5000": "5000",
  "6400": "6400",
  "8000": "8000",
  "10000": "10000",
  "12800": "12800",
  "16000": "16000",
  "20000": "20000",
  "25600": "25600",
  "Auto": "Auto",
};

pane.addBinding(object, "ISO Speed", { options });
pane.addBinding(object, "ISO Speed", { view: "list", options }); // doesn't work either
```

**Comment by cocopon** [comment]
This is caused by a bug of parsing color string.

https://github.com/cocopon/tweakpane/blob/55631021a0e27dfedff9fb72b3fb39abccf10206/packages/core/src/input-binding/color/converter/color-string.ts#L211

`'3200'` should not be parsed as a color string, but it is. I'll fix it later.

**Comment by cocopon** [comment]
Fixed, please wait for the next release.

**Comment by cocopon** [comment]
Released in 4.0.4. Thank you for reporting the issue.
